src/knex-tables.d.ts

```diff
@@ -190,6 +190,19 @@ declare module "knex/types/tables" {
     "id" | "subscriber_id" | "created_at"
   >;
 
+  interface SubscriberEmailPreferencesRow {
+    id: number;
+    subscriber_id: number;
+    instant_breach_alert: boolean;
+    all_emails_to_primary: boolean;
+    monthly_monitor_report: boolean;
+    monthly_monitor_report_at: Date | null;
+  }
+  type SubscriberEmailPreferencesAutoInsertedColumns = Extract<
+    keyof SubscriberEmailPreferencesRow,
+    "id" | "subscriber_id"
+  >;
```

Comment:
```suggestion
```

Looking at the definitions below, it doesn't look like you're actually using this type? (Which looking back honestly is probably the more readable approach, just explicitly excluding columns (i.e. `id`) rather than adding indirection through another type.)